astropy/time/tests/test_precision.py

```diff
@@ -15,6 +15,14 @@
 dt_tiny = TimeDelta(2. ** -52, format='jd')
 
 
+def test_abs_jd2_always_less_than_half():
+    """Make jd2 approach +/-0.5, and check that it doesn't go over."""
+    t1 = Time(2400000.5, [-2**-52, +2**-52], format='jd')
+    assert np.all(abs(t1.jd2) < 0.5)
+    t2 = Time(2400000., [0.5-2**-52, 0.5+2**-52], format='jd')
```

Comment:
Does this also need a check for the -0.5 case `[-0.5-2**-52, -0.5+2**-52]`?  I'm not sure if this ends up being a different case or not.

Reply:
I think it is the same, but, hey, doesn't cost anything to add it, so consider it done!